util/reggen/multi_register.py

```diff
@@ -106,18 +106,28 @@ def __init__(self,
             regs_per_creg = 1
 
         self.regs = []
-        creg_count = (self.count + regs_per_creg - 1) // regs_per_creg
-        for creg_idx in range(creg_count):
+        self.creg_count = (self.count + regs_per_creg - 1) // regs_per_creg
```

Comment:
This doesn't need to be a field: it's equal to `len(self.regs)`. Please change this back to a local.

Reply:
that makes sense. Removed, thanks